tests/backup/backup_restore_basic_test.go

```diff
@@ -3358,3 +3354,192 @@ var _ = Describe("{AlternateBackupBetweenNfsAndS3}", func() {
+		secondBackupName                    string
+		providers                           []string
+		restoreList                         []string
+		appNamespaces                       []string
+		sourceClusterRestoredNamespace      []string
+		destinationClusterRestoredNamespace []string
+		scheduledAppContexts                []*scheduler.Context
+		contexts                            []*scheduler.Context
+		appContexts                         []*scheduler.Context
+	)
+	backupLocationMap := make(map[string]string)
+	namespaceMapping := make(map[string]string)
+	sourceClusterNamespaceMapping := make(map[string]string)
+	destinationClusterNamespaceMapping := make(map[string]string)
+	restoredAppContexts := make([]*scheduler.Context, 0)
+	JustBeforeEach(func() {
+		StartTorpedoTest("BackupNamespaceInNfsRestoredFromS3", "Take a backup of namespace in NFS which is restored from s3 bucket or vice-versa", nil, 86089)
+		log.InfoD("Scheduling Applications")
+		scheduledAppContexts = make([]*scheduler.Context, 0)
+		for i := 0; i < 5; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				appNamespaces = append(appNamespaces, namespace)
+				scheduledAppContexts = append(scheduledAppContexts, ctx)
+			}
+		}
+		log.Infof("The list of namespaces deployed are", appNamespaces)
+		providers = getProviders()
+	})
+
+	It("Take a backup of namespace in NFS which is restored from s3 bucket or vice-versa", func() {
+		Step("Validate applications", func() {
+			log.InfoD("Validating applications")
+			ValidateApplications(scheduledAppContexts)
+		})
+
+		Step("Adding cloud credential and backup locations", func() {
+			log.InfoD("Adding cloud credential and backup locations")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, provider := range providers {
+				s3CloudCredName = fmt.Sprintf("%s-%s-%v", "cloudcred", provider, RandomString(5))
+				s3CloudCredUID = uuid.New()
+				firstBkpLocationName = fmt.Sprintf("%s-%s-%v-bl", provider, getGlobalBucketName(provider), RandomString(5))
+				firstBackupLocationUID = uuid.New()
+				err = CreateCloudCredential("aws", s3CloudCredName, s3CloudCredUID, orgID, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential:[%s] for org [%s] for s3 backup location", s3CloudCredName, orgID))
+				err = CreateBackupLocation(provider, firstBkpLocationName, firstBackupLocationUID, s3CloudCredName, s3CloudCredUID, getGlobalBucketName(provider), orgID, "")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating %s backup location: %s", provider, firstBkpLocationName))
+				backupLocationMap[firstBackupLocationUID] = firstBkpLocationName
+				if provider == "aws" {
```

Comment:
Change this to `provider != drivers.ProviderNfs`

Reply:
done